### PR TITLE
Exclude lib site-packages from overall coverage score

### PR DESCRIPTION
**Pull request by RaphaelMeudec**
At the moment, some site-packages are in the cov summary, which shouldn't be the case. This PR fixes the computation of coverage score.

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+### Summary
+
+### Related Issues
+
+### PR Overview
+
+- [ ] This PR requires new unit tests [y/n] (make sure tests are included)
+- [ ] This PR requires to update the documentation [y/n] (make sure the docs are up-to-date)
+- [ ] This PR is backwards compatible [y/n]
+- [ ] This PR changes the current API [y/n] (all API changes need to be approved by fchollet)
```
Comment (gabrieldemarmiesse) [inline PULL_REQUEST_TEMPLATE.md]: For the PR template, I think we're going to go with the new design which you can find here: https://github.com/keras-team/keras/pull/12169

Comment (RaphaelMeudec) [reply PULL_REQUEST_TEMPLATE.md]: @gabrieldemarmiesse Some sections don't appear when you open the pull request (those with shape ). Is it a wanted behaviour or should we fix this?

Comment (RaphaelMeudec) [reply PULL_REQUEST_TEMPLATE.md]: All the `<!-- XXX -->` are comments on Github and do not show up when opening a PR, but there are in the pull request template file. What should we do with them?

**Comment by gabrieldemarmiesse** [comment]
Thanks!

**Pull request merged**